src/app.js

```diff
@@ -1,21 +1,30 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+
+const createServer = http.createServer((req, res) => {
+  res.setHeader('Content-Type', 'text/html');
+  res.statusCode = 200;
+
+  const PORT = process.argv.PORT || 8080;
+
+  const myURL = new URL(req.url, `http://localhost:${PORT}`);
+  const parts = myURL.pathname.split('/').slice(1);
+  const query = {};
+
+  for (const [key, value] of myURL.searchParams.entries()) {
+    query[key] = value;
+  }
+
+  const response = {
+    parts,
+    query,
+  };
+
+  res.end(JSON.stringify(response));
+});
```

Comment:
don't see any sense to create this redundant variable 
```suggestion
  res.end(JSON.stringify({
    parts,
    query
  }));
});
```